### PR TITLE
feature/fix deprecated redis multi

### PR DESCRIPTION
**Pull request by maxim-erkin**

### CODE REVIEW

**Comment by jalkoby** [comment]
any updates?

**Comment by yurikoval** [comment]
@seanxiesx LGTM

File: lib/redis_rate_limiter.rb [hunk 1]
```diff
@@ -32,11 +32,11 @@ def initialize key, redis, options = {}
...
     subject_key = "#{@key}:#{subject}"
-    @redis.multi do
-      @redis.lpush(subject_key, time)
-      @redis.ltrim(subject_key, 0, @limit - 1)
-      @redis.expire(subject_key, @interval)
+    @redis.multi do |pipeline|
+      pipeline.lpush(subject_key, time)
+      pipeline.ltrim(subject_key, 0, @limit - 1)
     end
+    @redis.expire(subject_key, @interval)
```
Comment (seanxiesx) [inline lib/redis_rate_limiter.rb:39]: should we be moving this out of the multi block?

Comment (jalkoby) [reply lib/redis_rate_limiter.rb:39]: it was done to pass tests. but it definitely should be inside the block

Comment (seanxiesx) [reply lib/redis_rate_limiter.rb:39]: sounds good. happy to review an updated PR that keeps it within the block and handles the test issues.

**Comment by 5minpause** [comment]
@seanxiesx I updated the code to move the `expire` call into the block while keeping the tests green.

**Comment by seanxiesx** [comment]
> @seanxiesx I updated the code to move the `expire` call into the block while keeping the tests green.

nice. will you open a new PR? i should be able take a look and merge in the next few days.

**Comment by seanxiesx** [comment]
thanks @5minpause .
FYI, pushed 0.2.0 to rubygems

**Pull request merged**